modDesc.xml

```diff
@@ -35,6 +35,17 @@ or go directly to our <a href="https://github.com/Courseplay/Courseplay_FS22">Gi
 
 Many thanks to our translators and to our community for reporting bugs, giving us feedback and great ideas.
 
+Changelog 7.5.0.0:
+- New course generator 2.0 with new features and updated help menu.
+- A multitool course now contains 2-5 single course for each lane depending on the amount of vehicles you selected.
+- Added food mixer wagon, fertilizer/lime/manure sprayer and Göweil LT Master as valid target for shovel mode. 
+- Added chopper unloader support known from FS19 back in.
```

Comment:
Weil es in der deutschen Version steht:

(Premature calling of a second unloader is not available.)

Oder "Early calling"